spec/rubocop/cop/rails/pluralization_grammar_spec.rb

```diff
@@ -0,0 +1,97 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Rails::PluralizationGrammar do
+  subject(:cop) { described_class.new }
+  before(:each) do
+    inspect_source(cop, source)
+  end
+
+  shared_examples 'enforces pluralization grammar' do |method_name|
+    context "When #{method_name} is called on an unknown variable" do
+      context "when using the plural form ##{method_name}s" do
+        let(:source) { "some_variable.#{method_name}s" }
+        it 'does not register an offense' do
+          expect(cop.offenses).to be_empty
+        end
+      end
+
+      context "when using the singular form ##{method_name}" do
+        let(:source) { "some_method.#{method_name}" }
+
+        it 'does not register an offense' do
+          expect(cop.offenses).to be_empty
+        end
+      end
+    end
+
+    [1, 1.0].each do |singular_number|
+      context "when #{method_name} is called on a literal #{singular_number}" do
```

Comment:
This outer context does not make sense to me. It is immediately followed by the inverse making it confusing what is being tested. Possibly shorten this to `'singular literals'` or expanding the inner `context` to `"when calling the plural method #{method_name}s on a the singular #{singular_number}"`.